search/lib/api/wfs.js

```diff
@@ -177,16 +397,51 @@ async function getGranule (request, response) {
   logger.info(`GET /${providerId}/collections/${collectionId}/items/${conceptId}`);
   const event = request.apiGateway.event;
 
-  const cmrParams = {
-    provider_id: providerId,
-    concept_id: conceptId
-  };
+  //We need to make sure the granule belongs to the provider and the collection.
+  const cmrParams = Object.assign(
+    { concept_id: conceptId },
+    cmr.stacCollectionToCmrParams(providerId, collectionId)
+  );
```

Comment:
Matt, I changed this back because I think collectionId is still relevant. We don't want someone to manually type in a conceptId that doesn't belong to the collectionId and still return the result.

Reply:
Really good point, agree